info/v2/container.go

```diff
@@ -170,6 +170,8 @@ type ContainerStats struct {
 	PerfStats []v1.PerfStat `json:"perf_stats,omitempty"`
 	// Referenced memory
 	ReferencedMemory uint64 `json:"referenced_memory,omitempty"`
+	// Resource Control (resctrl) statistics.
```

Comment:
The rest of comments in `ContainerStats` struct do not have dot at the end, except `Timestamp` field